README.md

```diff
@@ -40,3 +40,9 @@ Please contribute to this repository with your questions or additions. To do so,
 | [Consent requirements](./ex-parte-renewals/consent.md)
 | [Worst case scenario challenges](./ex-parte-renewals/worst-case-scenario.md)
 
+
+## Public domain
+
+This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/) as indicated in [LICENSE](LICENSE).
```

Comment:
Will this `LICENSE` link work? Jekyll might not render it because it's not Markdown....

Let me clone this to my personal copy and check.

Reply:
Okay so on a test (Firefox) this link, on click, prompts a download of the `LICENSE` file. Do we want to add an extension (like `txt`)? That might make this a useful download.

Reply:
Done!

thank you for testing that! How would I have tested it?

Reply:
I have a personal fork of this repo that I have setup to use the same GitHub pages settings. I basically just replicated your change manually; I created a LICENSE file and then tried to visit the link.

Reply:
thanks for doing that work, I'm sorry it requires that much